Core/ConnectedShapeBasedInterpolation.h

```diff
@@ -130,7 +208,7 @@ class ConnectedShapeBasedInterpolation
 		}
 
 		std::vector<itk::Point<double, 2>> pts(num_objects);
-		for (int id=0; id<num_objects; ++id)
+		for (int id = 0; id < num_objects; ++id)
```

Comment:
why everywhere are int used instead of unsigned int? these are indices aren't they?